### DISCUSSION TITLE
Adding Ephemeris DE430 and DE431?

### DISCUSSION THREAD

**Discussion by rolomoto** [Stellarium/stellarium#1909] [Q&A] [answered]
I downloaded the two folders and placed them in ~Library/Application Support/Stellarium/ephem
but under ephemeris settings it still says unavailable, how can I get them installed?
Thanks

**Comment by gzotti** [comment, marked as answer]
You write "folders". Did you place the actual "files" there?
I have no Mac and cannot give more advice. Is this the actual Stellarium user directory on your system? (with config.ini etc.)

**Reply by AndiPangerangHasanuddin** [reply]
how about this?

<img width="686" alt="ephem stella_3" src="https://github.com/user-attachments/assets/2fa99d8f-b3bd-4110-88a8-7c4b1a76d76a">

<img width="691" alt="ephem stella_2" src="https://github.com/user-attachments/assets/0b19d70b-b10b-4585-8283-79431b2f243e">

<img width="1494" alt="ephem stella_1" src="https://github.com/user-attachments/assets/3e4fb2e1-55cf-493e-8987-eec09b961315">

could you help me solve this problem?

<img width="535" alt="ephem stella" src="https://github.com/user-attachments/assets/c57b2ef0-f9bd-44f9-bfd4-40ec98582b96">

thanks

**Reply by alex-w** [reply]
Stellarium expect to see files like `linux_p1550p2650.440` in `ephem` directory as it described in User Guide - not BSP!

**Comment by alex-w** [comment]
It works for me on the Mac. Which folders you are meant?

**Comment by rolomoto** [comment]
OK, thanks, I had to take the files out of the downloaded folders and place them in them in the ephem folder.
